app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = []
+    for person in people:
+        result += {Person(person.get("name"), person.get("age"))}
+    for i, person in enumerate(people):
+        if "wife" in person and person.get("wife") is not None:
```

Comment:
```suggestion
        if person.get("wife") is not None:
```
The same with husband